src/main/java/org/ice4j/ice/Component.java

```diff
@@ -599,13 +599,22 @@ public String toString()
             buff.append("\n")
                 .append(remoteCandidatesCount)
                 .append(" Remote candidates:");
-            buff.append("\ndefault remote candidate: ")
-                .append(getDefaultRemoteCandidate());
+            Candidate<?> defaultRemoteCandidate = getDefaultRemoteCandidate();
+            buff.append("\ndefault remote candidate: ");
+            if (defaultRemoteCandidate != null)
+            {
+                buff.append(getDefaultRemoteCandidate().toRedactedString());
+            }
+            else
+            {
+                buff.append((String)null);
```

Comment:
I think `append("null")` would be clearer, I wasn't sure what it does with an actuall `null`